doc/installing/universal.md

```diff
@@ -102,6 +103,16 @@ $ sudo -H /usr/bin/python3 install.py server
 The installation script will automatically create a user called `openquake` and will install the engine in the directory `/opt/openquake`.
 Calculation data will be stored in `/var/lib/openquake/oqdata`.
 
+## `devel_server` installation
+
+This method adds to `server` installation the possibility to modify the engine codebase or add new features for the engine.
```

Comment:
~the possibility to modify the engine codebase or add new features for the engine.~ => the possibility to run the engine from a git repository.